src/data/boss.json

```diff
@@ -163,6 +163,98 @@
         "wikiLink": "https://escapefromtarkov.fandom.com/wiki/knight",
         "behavior": "Rush"
     },
+    {
+        "name": "Big Pipe",
+        "normalizedName": "big-pipe",
+        "details": "The leader of 'The Goons'. Can spawn on many different maps.",
```

Comment:
According to the Wiki, Knight is the leader.

Reply:
i can change that. i didnt even realize i didnt change that on my end. Sorry! i will change that now